sim/dex-data.ts

```diff
@@ -187,6 +187,20 @@ export class RuleTable extends Map<string, string> {
 		return this.has(`*${thing}`);
 	}
 
+	isRestrictedSpecies(species: Species) {
+		if (this.has(`*pokemon:${species.id}`)) return true;
+		if (this.has(`+pokemon:${species.id}`)) return false;
```

Comment:
Aren't these all backwards? Shouldn't the `+` check come first?

Reply:
I can swap them over to make more sense to you, but it makes no difference because you can't have both versions in the same rule table at once.

Reply:
I think there might be complicated interactions with subrules that let it happen. Either way, there's nothing in the type system disallowing it.